### PR TITLE
Change style of bouquet to me more consistent with Lunaria and NFT Sender

### PR DESCRIPTION
**Pull request by 0xjimmy**
- Updated buttons
- Added Footer
- Changed Navbar Design
 
Issues and styling for each section will be addressed in separate PR's and then merged into main once all are complete.

image
image
image

### CODE REVIEW

**Review by KillariDev — APPROVED** [review]

**Comment by 0xjimmy** [comment]
Going to make some more PR's to the style branch. Then thoroughly test again before merging into main.
- [ ] Settings input values don't update to match current settings on open and reset click
- [ ] Add back Configure section and fix error with caching wrong private key inputs
- [ ] Implement refund from burner / new burner
- [ ] Submit UX + show links to confirmed transactions on Etherscan on complete

**Comment by 0xjimmy** [comment]
Appears to be issue when connecting sometimes with just Metamask (Or rather no Interceptor), need to fix and test to find any other small bugs before merging into main

**Comment by 0xjimmy** [comment]
This PR has a decent amount of changes, most have been reviewed already in small PR's into this branch, but I think some eyes on this before merging to main may be a good idea.

**Review by KillariDev — APPROVED** [review]

File: app/ts/components/Submit.tsx [hunk 1]
```diff
@@ -17,7 +19,13 @@ type PendingBundle = {
 			included: boolean
 		}
 	}
-	error?: Error
+	error?: Error,
+	success?: {
```
[the remark this answers is not in the document]
Comment (0xjimmy) [reply app/ts/components/Submit.tsx:23]: I tried change types earlier, but found that I would need to change a lot of how I checked the data too. So its new issue https://github.com/DarkFlorist/bouquet/issues/64

**Pull request merged**